ktor-server/ktor-server-plugins/ktor-server-i18n/build.gradle.kts

```diff
@@ -7,3 +7,13 @@ description = "I18n support to Ktor"
 plugins {
     id("ktorbuild.project.server-plugin")
 }
+
+kotlin {
+    sourceSets {
+        jvmTest {
+            dependencies {
+                implementation(project(":ktor-server:ktor-server-plugins:ktor-server-status-pages"))
+            }
+        }
```

Comment:
@coderabbitai, remember that syntax `jvmTest.dependencies { ... }` is preferred over `jvmTest { dependencies { ... } }` as it gives the level of nesting lower by one.